src/store/modules/util/index.ts

```diff
@@ -21,7 +21,8 @@ const utilModule: Module<UtilState, RootState> = {
     productStores: [],
     facilities: [],
     shipmentGatewayConfigs: [],
-    isForceScanEnabled: false
+    isForceScanEnabled: false,
+    barcodeIdentificationPref: ""
```

Comment:
As we have defined `internalName` as default value for barcodeIdentifier we can have the same value here in state as its default declaration